src/RA_Dlg_MemBookmark.cpp

```diff
@@ -888,15 +848,8 @@ void Dlg_MemBookmark::OnLoad_NewRom()
     if (::GetDlgItem(m_hMemBookmarkDialog, IDC_RA_LBX_ADDRESSES) != nullptr)
     {
         ClearAllBookmarks();
-
-        std::wstring file;
-        {
-            const auto& pGameContext = ra::services::ServiceLocator::Get<ra::data::GameContext>();
-            std::wostringstream oss;
-            oss << g_sHomeDir << RA_DIR_BOOKMARKS << pGameContext.GameId() << L"-Bookmarks.txt";
-            file = oss.str();
-        }
-        ImportFromFile(file);
+        const auto& pGameContext = ra::services::ServiceLocator::Get<ra::data::GameContext>();
+        ImportFromFile(ra::StringPrintf(L"%s%s%u-Bookmarks.txt", g_sHomeDir, RA_DIR_BOOKMARKS, pGameContext.GameId()));
```

Comment:
A function return is a rvalue-reference. Originally ImportFromFile used copying which is slow. const ref caused problems, moving is always better.